app/views/admin/events/_action_page_events.html.erb

```diff
@@ -51,34 +53,6 @@
           end %>
         <% end %>
 
-        <% if @actionPage.enable_call? && @actionPage.call_campaign %>
-          <h3>Call Tool</h3>
-          <%= line_chart(admin_action_page_events_path(@actionPage, type: :calls),
-                         height: "200px", discrete: true) %>
-        <% end %>
-
-        <% if @actionPage.enable_tweet? && @actionPage.tweet %>
-          <h3>Tweet Tool</h3>
-          <%= line_chart(admin_action_page_events_path(@actionPage, type: :tweets),
-                         height: "200px", discrete: true) %>
-        <% end %>
-
-        <% if @actionPage.enable_petition? && @actionPage.petition && @actionPage.petition.persisted? %>
-          <h3 class=clearfix>
-            Petition Tool
-            <% if can?(:export_signatures) %>
```

Comment:
I've removed this permissions check. It falls through to the default check that the user is an admin, which they need to be to view this page. We're not using this permissions system extensively and the next link over lets them view the full list of signatures anyway.